rpc/api.proto

```diff
@@ -309,7 +310,13 @@ message GetTicketsResponse {
 		}
 		TicketStatus ticket_status = 3;
 	}
+	message BlockMeta {
```

Comment:
I probably wouldn't name it BlockMeta (as it comes from the udb.BlockMeta name which is being removed in favor of just working with the headers directly).  BlockDetails works better I think.

Reply:
There's a global `BlockDetails` message. You don't mind having both a nested and a global `BlockDetails` type? Alternative names having a duplicate name is not ok: BlockHeader, BlockHeaderInfo...?

Reply:
I don't think the nested type would cause any issues even in languages without support for nested types (e.g. Go).  In the Go bindings, this would become GetTicketsResponse_BlockDetails.  I suppose there could be some confusion and possibly bugs as well if you are working in a language without static typing.